tests/data/test_VideoDataset.py

```diff
@@ -192,11 +192,13 @@ def test_video_dataset_no_read_rights(self):
                     os.chmod(filepath, 0o000)
             # This will not raise any Permissions error, as they are caught by torchvision:
             # https://github.com/pytorch/vision/blob/5985504cc32011fbd4312600b4492d8ae0dd13b4/torchvision/io/video.py#L397
-            with warnings.catch_warnings(record=True) as caught_warning:
-                dataset = LightlyDataset(self.input_dir)
-            expected_warning = "Caught error: [Errno 13] Permission denied:"
-            has_warning = [True for warning in caught_warning if expected_warning in str(warning)]
-            self.assertEqual(len(has_warning), n_videos)
+            # We cannot count warnings, since these are emitted only from torchvision version 0.11 onwards
```

Comment:
Why do you comment out this code instead of deleting it? If we want to have it again, we can simply go to an older commit.

Reply:
Fair.

Commented lines removed and clean.